Dockerfile.template

```diff
@@ -9,7 +9,7 @@ FROM balenalib/%%BALENA_MACHINE_NAME%%-python:3.8
 COPY --from=build /usr/local/bin/uwsgi /usr/src/linux-kbuild-5.10/scripts/sign-file /usr/local/bin/
 COPY --from=build /usr/local/lib/python3.8/site-packages/ /usr/local/lib/python3.8/
 
-RUN install_packages cryptsetup efitools gnupg2 libxml2 sbsigntool
+RUN install_packages cryptsetup efitools gnupg2 libxml2 sbsigntool openssh-client
```

Comment:
Is the `openssh-client` only there to calculate the fingerprint? Is this something that the Pi needs or is it just to find a name for the key? The GPG API uses fingerprints because that is how GPG identifies keys, I don't think we need to replicate that for RSA unless necessary.

Reply:
You're right , there's no need to read the fingerprint. I'll remove it.